web/modules/custom/hoeringsportal_audit_log/src/EventSubscriber/ControllerListener.php

```diff
@@ -40,10 +40,19 @@ public function __construct(
    */
   public function onController(ControllerEvent $event): void {
     $pathInfo = $event->getRequest()->getPathInfo();
+    $requestUri = $event->getRequest()->getRequestUri();
+    $urlPatterns = $this->configHelper->getUrlPattern();
+
+    foreach ($urlPatterns as $urlPattern) {
+      if (preg_match($urlPattern, $requestUri)) {
```

Comment:
You may want to [suppress any errors](https://www.php.net/manual/en/language.operators.errorcontrol.php) (in the pattern) here … 